src/community/features-templating/features-templating-core/src/test/java/org/geoserver/featurestemplating/builders/impl/DynamicIncludeFlatBuilderTest.java

```diff
@@ -0,0 +1,126 @@
+package org.geoserver.featurestemplating.builders.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.fasterxml.jackson.core.JsonEncoding;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import net.sf.json.JSONObject;
+import net.sf.json.JSONSerializer;
+import org.geoserver.featurestemplating.configuration.TemplateIdentifier;
+import org.geoserver.featurestemplating.readers.RecursiveJSONParser;
+import org.geoserver.featurestemplating.writers.GeoJSONWriter;
+import org.geoserver.platform.resource.FileSystemResourceStore;
+import org.geotools.data.DataTestCase;
+import org.geotools.feature.simple.SimpleFeatureBuilder;
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder;
+import org.geotools.jdbc.JDBCDataStore;
+import org.junit.Before;
+import org.junit.Test;
+import org.opengis.feature.Feature;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.xml.sax.helpers.NamespaceSupport;
+
+public class DynamicIncludeFlatBuilderTest extends DataTestCase {
+
+    private static final String BASE =
```

Comment:
no merge here, it is an inclusion this should not be needed.

Reply:
removed BASE string and other unused variable after deleting that.